2025/social-wg.html

```diff
@@ -366,15 +366,9 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             </dd>
             <dt><a href="https://www.w3.org/groups/cg/socialcg/">Social Web Incubator Community Group</a></dt>
             <dd><p>The SocialCG provides space to collaborate and coordinate for implementors who are building on any of the specifications published by the Social Web WG, and related technologies. It is also a place to incubate new proposals which build on or complement the Social Web WG recommendations.</p>
-                      <p>
-          This group is expected to coordinate with the
-          <a href="https://www.w3.org/groups/cg/socialcg/">Social Web Incubator Community Group</a>
-          on consensus-based proposals related to content changes for the
-          Social Web Working Group Deliverables.
-          The Chairs of this group should reject proposals that are incompatible
-          with this Charter.
-        </p>
-
+            <p>
+              The Working Group is expected to request reviews from the Community Group of all of the Working Group deliverables.
```

Comment:
```suggestion
              The Working Group will request reviews from the Community Group of all WG deliverables.
```

minor edit to clarify stronger intent and simplify wording.

Reply:
we did discuss `deliverables` vs `Deliverables` and we wanted to keep that a little more informal to cast as wide a net as possible for what we send to CG review. (like what if something is an edge case not covered exactly by Section 3 aka Deliverables)

Reply:
re "will" that works for me